.github/workflows/lint-and-test.yml

```diff
@@ -18,9 +18,9 @@ jobs:
         uses: azure/setup-helm@v1
 
       - name: Set up python
-        uses: actions/setup-python@v3
+        uses: actions/setup-python@v4
```

Comment:
There is no breaking changes. :eyes:
https://github.com/actions/setup-python/releases/tag/v4.0.0